### DISCUSSION TITLE
Highlighting (animation) for targeted headlines

### DISCUSSION THREAD

**Discussion by JoCa96** [SchwarzIT/onyx#616] [Ideas]
A headline can be targeted via the hash of a URL (like example.com/__#headline1__). This can be achieved by opening a corresponding link or clicking a link with an anchor (which will scroll to an element) inside the same page.

On pages with a lot of text, it is hard to discern which element is targeted. This is made even harder by the fact that different browsers and web apps might define different scroll margins.

We can help the user by either statically highlighting a target headline or animating the headline:

https://github.com/SchwarzIT/onyx/assets/18483883/9929638a-a112-40ac-9e4f-afc18e2e73df

See also: https://developer.mozilla.org/en-US/docs/Web/CSS/:target

**Comment by mj-hof** [comment]
Maybe a cool feature for our showcase/documentation
